rules/windows/process_creation/proc_creation_win_more_vbc_lummac.yml

```diff
@@ -0,0 +1,28 @@
+title: Detect The Execution Of More.com And Vbc.exe Related to Lummac Stealer
+id: 19b3806e-46f2-4b4c-9337-e3d8653245ea
+status: test
+description: Detects the execution of more.com and vbc.exe in the process tree. This behavior was observed by a set of samples related to Lummac Stealer. The Lummac payload is injected into the vbc.exe process.
+references:
+    - https://www.virustotal.com/gui/search/behaviour_processes%253A%2522C%253A%255C%255CWindows%255C%255CSysWOW64%255C%255Cmore.com%2522%2520behaviour_processes%253A%2522C%253A%255C%255CWindows%255C%255CMicrosoft.NET%255C%255CFramework%255C%255Cv4.0.30319%255C%255Cvbc.exe%2522/files
```

Comment:
It requires a paid account and therefore not public.
I find this https://www.joesandbox.com/analysis/1554788/0/html